packages/dds/matrix/src/runtime.ts

```diff
@@ -13,6 +13,9 @@ import {
 import { pkgVersion } from "./packageVersion";
 import { SharedMatrix } from "./matrix";
 
+/**
+ * @public
```

Comment:
While we're here
```suggestion
 * {@link @fluidframework/datastore-definitions#IChannelFactory} for {@link ISharedMatrix}.
 * @public
```

Reply:
@Josmithr There wasn't an export called ISharedMatrix, but I think you meant SharedMatrix. I made the update!